lib/foundation_rails_helper/flash_helper.rb

```diff
@@ -18,29 +18,29 @@ module FlashHelper
       :warning   => :warning,
       :primary   => :primary
     }
-    def display_flash_messages(key_matching = {})
-      key_matching = DEFAULT_KEY_MATCHING.merge(key_matching)
+    def display_flash_messages(**options)
```

Comment:
I like the second option using one options hash.  That way, we can easily add more options, if need be, besides `closable`.